Package.swift

```diff
@@ -20,7 +20,7 @@ let package = Package(
   targets: [
     .target(
       name: "AsyncAlgorithms",
-      dependencies: [.product(name: "Collections", package: "swift-collections")],
+      dependencies: [.product(name: "OrderedCollections", package: "swift-collections")],
```

Comment:
I don't think this is enough. We also need to depend on `DequeModule` since we use `Deque` in quite a few places

Reply:
Ah I didn't realize, added in d685a5086ccb9decfa6dc83a8558c5c5911559b0